bin/p2-bootstrap/bootstrap.go

```diff
@@ -102,7 +103,9 @@ func VerifyConsulUp(timeout string) error {
 		return nil
 	}
 
-	store := kp.NewStore(kp.Options{})
+	store := kp.NewStore(kp.Options{
+		Token: *consulToken, // not actually necessary because this endpoint is unauthenticated
```

Comment:
No the catalog endpoints actually cannot be authenticated, consul doesn't have that feature. kp contains primarily kv endpoints (which support ACLs) but it also contains service endpoints for service registration (no ACL support) and catalog endpoints for pinging (no ACL support).